docs/sources/flow/reference/components/loki.source.syslog.md

```diff
@@ -0,0 +1,143 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/loki.source.syslog
+title: loki.source.syslog
+---
+
+# loki.source.syslog
+
+`loki.source.syslog` listens for syslog messages over TCP or UDP connections
+and forwards them to other `loki.*` components. The messages must be compliant
+with the [RFC5424](https://www.rfc-editor.org/rfc/rfc5424) format.
+
+The component starts a new syslog listener for each of the given `config`
+blocks and fans out incoming entries to the list of receivers in `forward_to`.
+
+Multiple `loki.source.syslog` components can be specified by giving them
+different labels.
+
+## Usage
+
+```river
+loki.source.syslog "LABEL" {
+  listener {
+    address = "LISTEN_ADDRESS"
+  }
+  ...
+
+  forward_to = RECEIVER_LIST
+}
+```
+
+## Arguments
+
+`loki.source.syslog` supports the following arguments:
+
+Name         | Type                   | Description          | Default | Required
+------------ | ---------------------- | -------------------- | ------- | --------
+`forward_to` | `list(LogsReceiver)`   | List of receivers to send log entries to. | | yes
+
+## Blocks
+
+The following blocks are supported inside the definition of
+`loki.source.syslog`:
+
+Hierarchy | Name | Description | Required
+--------- | ---- | ----------- | --------
+listener | [config][] | Configures a listener for IETF Syslog (RFC5424) messages. | no
+listener > tls_config | [tls_config][] | Configures TLS settings for connecting to the endpoint for TCP connections. | no
+
+The `>` symbol indicates deeper levels of nesting. For example, `config > tls_config` 
+refers to a `tls_config` block defined inside a `config` block.
+
+[listener]: #listener-block
+[tls_config]: #tls_config-block
+
+### listener block
+
+The `listener` block defines the listen address and protocol where the listener
+expects syslog messages to be sent to, as well as its behavior when receiving
+messages.
+
+The following arguments can be used to configure a `listener`. Only the
+`address` field is required and any omitted fields take their default
+values.
+
+Name                     | Type          | Description | Default | Required
+------------------------ | ------------- | ----------- | ------- | --------
+`address`                | `string`      | The `<host:port>` address to listen to for syslog messages. | | yes
```

Comment:
Should be using placeholder syntax here (so people don't think `<>` is something they need to type):

```suggestion
`address`                | `string`      | The `HOST:PORT` address to listen to for syslog messages. | | yes
```

Reply:
Thanks for the catch!